README.md

```diff
@@ -73,6 +73,7 @@ Options:
                          The repository name to create the Nullify issue dashboard in e.g. cli
   --header HEADER        List of headers for the DAST agent to authenticate with your API
   --local                Test the given app locally for bugs and vulnerabilities in private networks
+  --version VERSION      Version of the DAST local image that is used for scanning [default: 0.0.0]
```

Comment:
should we default to `latest` instead?

Reply:
https://github.com/Nullify-Platform/dast-local/pull/17 should fix to create latest